docs/en/sql-reference/functions/array-functions.md

```diff
@@ -176,6 +176,52 @@ hasAny(array1, array2)
 
 `SELECT hasAll([[1, 2], [3, 4]], [[1, 2], [1, 2]])` returns `1`.
 
+## hasSubStr {#hassubseq}
+
+Checks whether all the elements of array2 appear in array1 in the same exact order.
```

Comment:
Better. And yet I think it is also important to stress: array1 = prefix + array2 + suffix

Reply:
I'll make the change.